pyOCD/interface/hidapi_backend.py

```diff
@@ -107,3 +107,7 @@ def close(self):
         """
         logging.debug("closing interface")
         self.device.close()
+
+    def setPacketCount(self, count):
+        # No interface level restrictions on count
+        self.packet_count = count
```

Comment:
This probably won't increase speed much with just this.  As is, the hidapi backend only reads or writes, but doesn't do both at the same time.  If a reader thread was created then this would likely double speed.  An easy way to check the effect on performance is to run speed_test.py.